src/ContextMenu.js

```diff
@@ -98,7 +98,7 @@ export default class ContextMenu extends Component {
     }
 
     handleHide = (e) => {
-        if (this.state.isVisible) {
+        if (this.state.isVisible && (e.detail === undefined || e.detail.id === undefined || e.detail.id === _this.props.id)) {
```

Comment:
Can you change this to?
``` js
if (this.state.isVisible && (!e.detail || !e.detail.id || e.detail.id === this.props.id))
```
1. It will handle even `null`
2. typo of `this`